src/sql/parts/notebook/cellViews/code.css

```diff
@@ -13,3 +13,9 @@ code-component .toolbar {
 	border-right-width: 1px;
 	border-right-style: solid;
 }
+
+code-component .toolbarIconRun {
+	height: 20px;
+	background-image: url('../../../base/browser/ui/taskbar/media/start.svg');
```

Comment:
Do we use relative URLs in other places? Please verify this is the correct way to reference things.

Reply:
Yes, we do. Need to check with Raj to see where we've landed on our own icons.

Reply:
It's under notebook -> media